README.md

```diff
@@ -272,3 +272,9 @@ The backend driver is based on the implementation of [uber/athenadriver](https:/
 - Configure and use [Templates and variables](https://grafana.com/docs/grafana/latest/variables/).
 - Add [Transformations](https://grafana.com/docs/grafana/latest/panels/transformations/).
 - Set up alerting; refer to [Alerts overview](https://grafana.com/docs/grafana/latest/alerting/).
+
+## Async Query Data Support
+
+Async Query Data support enables an asynchronous query handling flow. With Async Query Data support enabled, queries will be handled over multiple requests (starting, checking its status, and fetching the results) instead of having a query be started and resolved over a single request. This is useful for queries that can potentially run for a long time and timeout.
```

Comment:
might be worth having the docs team check this out if they have time!

Reply:
@grafana/docs-squad can someone review this if you have the time please?